src/pandasaurus/utils/query_utils.py

```diff
@@ -0,0 +1,71 @@
+from ..config import default_config
+from typing import List
+from SPARQLWrapper import SPARQLWrapper, JSON
```

Comment:
Consider using OAK as a wrapper for querying UberGraph.  There are some advantages  - e.g. support for standard prefixes.  @anitacaron  can tell you more.

Reply:
Even thought I'm not sure how the prefix validation will be done, do we really need an excavator for a task that can be done with a small shovel?